.eslintrc.json

```diff
@@ -1,19 +1,34 @@
 {
-	"extends": ["eslint:recommended", "prettier"],
 	"env": {
 		"es6": true,
 		"node": true
 	},
+	"extends": [
+		"eslint:recommended",
+		"prettier",
```

Comment:
The two `prettier` extends need to be the final ones. That way, they can overwrite anything that comes before them.